modules/flyte/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "info" {
+  description = "Flyte info"
+  value = {
+    eks = {
```

Comment:
Why does do we need this have a parent `eks`?

Reply:
We wanted it to have a similar format to the domino config, which seems to have solidified around a similar level of nesting: https://github.com/cerebrotech/domino-config/blob/f3fc3f1af3520fee60f0d9aeb5f4c5df1b8237ae/domino_config/types1_7.py#L51-L52. Is this an issue that blocks the PR?